flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -524,6 +524,28 @@ public boolean isForceCheckpointing() {
 		return checkpointCfg.isForceCheckpointing();
 	}
 
+	/**
+	 * Returns whether Unaligned Checkpoints are force-enabled.
+	 *
+	 * @deprecated Forcing Unaligned Checkpoints will be removed in future version.
+	 */
+	@Deprecated
+	@PublicEvolving
+	public boolean isUnalignedCheckpointsEnabled() {
+		return checkpointCfg.isUnalignedCheckpointsEnabled();
+	}
+
+	/**
+	 * Returns whether Unaligned Checkpoints are force-enabled.
+	 *
+	 * @deprecated Forcing checkpoints will be removed in future version.
+	 */
+	@Deprecated
```

Comment:
I'd not deprecate it at this point. I see it decoupled from the deprecation of forcing checkpoints.